src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineController.java

```diff
@@ -38,26 +38,47 @@
     /**
      * Does a sideline exist in the started state?
      * @param sidelineRequest sideline request.
-     * @return true it does, false it does not.
+     * @return true it is in this state, false if it is not.
      */
-    boolean isSidelineStarted(SidelineRequest sidelineRequest);
+    boolean isPaused(SidelineRequest sidelineRequest);
 
     /**
      * Start sidelining.
      * @param request Sideline request, container an id and a filter chain step.
      */
-    void startSidelining(final SidelineRequest request);
+    void pause(final SidelineRequest request);
 
     /**
-     * Does a sideline exist in the stopped state?
+     *
      * @param sidelineRequest sideline request.
-     * @return true it has, false it has not.
+     * @return true it is in this state, false if it is not.
      */
-    boolean isSidelineStopped(SidelineRequest sidelineRequest);
+    boolean isResumed(SidelineRequest sidelineRequest);
 
     /**
-     * Stop sidelining.
+     *
+     * @param sidelineRequest
+     */
+    void resume(final SidelineRequest sidelineRequest);
+
+    /**
+     * Does a sideline exist in the finished state?
```

Comment:
I agree with peek, we should figure out a better name.

Reply:
I’m open to suggestions.

Reply:
Bounded Completed Something Or oTher. 

I have no good suggestions :/